lib/eco/autolboxdetector.py

```diff
@@ -78,7 +78,7 @@ def detect_lbox(self, errornode):
 
         maxdistance = {}
         # find max distance by startnode
-        for start, end, lang, dist in valid:
+        for start, _, _, dist in valid:
```

Comment:
ISTR there was another one of these a few lines below that's probably worth the same treatment.

Reply:
I think that one uses all elements of the tuple.

Reply:
Ah OK.